### PR TITLE
[Copy] Update notifications pane text and null message

### LINKED ISSUE #10639: 🐛 Update notifications pane text and `null` state message
## 🐛 Bug

The null state message for the notifications pane includes references to features that do not exist. And the notification pane copy could be better for users implying assistive technologies.

## 📋 Steps to Reproduce

1. Navigate to notifications page
2. Observe copy that does states "As you receive notifications, they'll appear here and automatically be categorized on whether you've actioned or pinned them.

## 🙋‍♀️ Proposed Solution

Implement copy as in this screenshot please:

Image

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the bug was addressed.

- [ ] copy updated in pane
- [ ] copy updated in null state

### PR DESCRIPTION
**Pull request by yonikid15**
🤖 Resolves #10639 

## 👋 Introduction

- Updates the notifications pane text and null message

## 🧪 Testing

Assist reviewers with steps they can take to test that the PR does what it says it does.

1. Login
2. Open up notifications pane and ensure text and null message is correct.

## 📸 Screenshot

image

### CODE REVIEW

**Review by vd1992 — DISMISSED** [review]
👍 

image

File: apps/web/src/components/NotificationDialog/NotificationDialog.tsx [hunk 1]
```diff
@@ -146,8 +146,8 @@ const NotificationDialog = () => {
                     {intl.formatMessage(
                       {
                         defaultMessage:
-                          "Welcome to your notification panel. Click or tap a notification to be taken to the relevant page. Use the <icon></icon> icon to mark a specific notification as read, pin it, or delete it.",
```
Comment (vd1992) [inline apps/web/src/components/NotificationDialog/NotificationDialog.tsx:149]: Having removed the icon, don't think you need the circled bit anymore

image

**Review by vd1992 — CHANGES_REQUESTED** [review]
One thing

**Comment by petertgiles** [comment]
Vachan has merged the fix for the overlay.  Could you please merge main?

File: apps/web/src/components/NotificationDialog/NotificationDialog.tsx (hunk 1 above)
Comment (yonikid15) [reply apps/web/src/components/NotificationDialog/NotificationDialog.tsx]: Nice catch! Fixed by [0d08784](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10853/commits/0d087840cf042fb15c721b2fdcc17cb16cab4384)

**Review by vd1992 — APPROVED** [review]
Just check that little bit of linting first and then it's good

**Pull request merged**